### PR TITLE
Doc: document update base on release_v1.5

### PR DESCRIPTION
**Pull request by fuzhongl**
Signed-off-by: fuzhongl <fuzhong.liu@intel.com>

### CODE REVIEW

File: doc/getting-started/rt_industry.rst [hunk 1]
```diff
@@ -6,8 +6,8 @@ Getting Started Guide for ACRN Industry Scenario
 Verified version
 ****************
 
-- Clear Linux version: **31670**
-- ACRN-hypervisor tag: **v1.4**
+- Clear Linux version: **32030**
+- ACRN-hypervisor tag: **v1.5**
```
Comment (NanlinXie) [inline doc/getting-started/rt_industry.rst]: @fuzhongl , Please modify as  v1.5(acrn-2020w01.1-140000p)

Comment (fuzhongl) [reply doc/getting-started/rt_industry.rst]: Done!

**Comment by wenlingz** [comment]
@fuzhongl please fix the build fail, thanks
make[1]: Entering directory '/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/doc'

New errors/warnings found, please fix them:
==============================================

/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:34: WARNING: Title level inconsistent:

Build ACRN EFI Images
=====================
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:57: WARNING: Title level inconsistent:

Build Service VM Kernel
=======================
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:70: WARNING: Title level inconsistent:

Update Kernel on KBL NUC
========================
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:91: WARNING: Title level inconsistent:

Create Windows 10 Image
=======================
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:100: WARNING: Title level inconsistent:

Preparations
------------
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:126: WARNING: Title level inconsistent:

Install Windows 10 ADK
----------------------
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:143: WARNING: Title level inconsistent:

Pre-install drivers and re-generate Windows ISO
-----------------------------------------------
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:311: WARNING: Title level inconsistent:

Create Raw Disk
---------------
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:320: WARNING: Title level inconsistent:

Install Windows 10
------------------
/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/build/doc/rst/tutorials/using_windows_as_uos.rst:406: WARNING: Title level inconsistent:

Boot Windows with GVT-g on ACRN
===============================

make[1]: Leaving directory '/root/workspace/acrn-hypervisor-premerge-build-image/acrn-hypervisor/doc'
make[1]: *** [Makefile:61: html] Error 1
make: *** [Makefile:159: doc] Error 2
Build step 'Execute shell' marked build as failure

File: doc/tutorials/acrn_ootb.rst [hunk 2]
```diff
@@ -88,7 +88,7 @@ Step 1: Create a Service VM YAML file and script
            {cmd: "${yamlDir}/service-os-post.sh ${chrootDir}"},
         ]
 
-        version: 31670
+        version: 3203/0
```
Comment (NanlinXie) [inline doc/tutorials/acrn_ootb.rst]: Should be 32030

Comment (fuzhongl) [reply doc/tutorials/acrn_ootb.rst]: Done!

File: doc/getting-started/rt_industry.rst [hunk 3]
```diff
@@ -6,8 +6,8 @@ Getting Started Guide for ACRN Industry Scenario
 Verified version
 ****************
 
-- Clear Linux version: **31670**
-- ACRN-hypervisor tag: **v1.4**
+- Clear Linux version: **32030**
+- ACRN-hypervisor tag: **v1.5(acrn-2020w01.1-140000p)**
```
[the remark this answers is not in the document]
Comment (fuzhongl) [reply doc/getting-started/rt_industry.rst]: Done!

File: doc/getting-started/rt_industry.rst [hunk 4]
```diff
@@ -6,8 +6,8 @@ Getting Started Guide for ACRN Industry Scenario
...
 - ACRN-Kernel(Service VM kernel): **4.19.78-98.iot-lts2018-sos**
```
[the remark this answers is not in the document]
Comment (fuzhongl) [reply doc/getting-started/rt_industry.rst]: Done!

File: doc/tutorials/using_windows_as_uos.rst [hunk 5]
```diff
@@ -14,6 +15,12 @@ The following Intel Kaby Lake NUCs are verified:
    "NUC7i7DNHE", "DNKBLi7v.86A.0052.2018.0808.1344", "`link <https://downloadcenter.intel.com/download/28886?v=t>`__"
    "NUC7i5DNHE", "DNKBLi5v.86A.0060.2018.1220.1536", "`link <https://downloadcenter.intel.com/download/28885?v=t>`__"
 
+Verified version
+****************
+#. Clear Linux version: 32030
+#. ACRN-hypervisor tag: v1.5(acrn-2020w01.1-140000p)
```
[the remark this answers is not in the document]
Comment (fuzhongl) [reply doc/tutorials/using_windows_as_uos.rst]: Done!

**Pull request merged**